src/common/hooks/useAtomWithPrevent.ts

```diff
@@ -82,17 +83,22 @@ export function useAtomWithPrevent<T extends Entity>(
       const currentEntityPaths = generatePaths(entity as T);
 
       const currentPathsForExcluding = currentEntityPaths.filter((path) =>
-        EXCLUDING_PROPERTIES_KEYS.some((excludingPropertyKey) =>
-          path.includes(excludingPropertyKey)
+        EXCLUDING_PROPERTIES_KEYS.some(
+          (excludingPropertyKey) =>
+            path?.includes(excludingPropertyKey) ||
+            (path?.includes('line_items') && path?.split('.')?.[2] === '_id')
         )
       );
 
       const updatedEntity = cloneDeep(entity) as T;
 
       currentPathsForExcluding.forEach((path) => {
-        if (!path.includes('.')) {
-          delete updatedEntity[path as unknown as keyof Entity];
-          delete currentInitialValue[path as unknown as keyof Entity];
+        if (
+          !path?.includes('.') ||
+          (path?.includes('line_items') && path?.split('.')?.[2] === '_id')
```

Comment:
Ugh, please explain change on this and logic behind it, seems weird. 

Also, please add it as comment for future reference.

Reply:
@beganovich Sure, the comment has been added. This logic aims to exclude unnecessary properties from tracking. 

The `_id` property for each line item is newly generated whenever we load the edit page, eliminating the need to track them at all. Paths are generated conventionally - properties on the first level of an object use just the property name, while nested properties use dots between levels. For example, a path like `'line_items.0._id'` indicates that if the path includes `'line_items'` and the property is `'_id'`, we exclude that path from tracking. This logic will only exclude `_id` properties from line items since it specifically targets these paths. All other line item properties will be tracked normally.